docs/get-started/v2/get-started/features/noise-cancellation.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Noise Cancellation
+nav: 3.9
+---
+
+The Noise Cancellation feature is an invaluable tool designed to enhance the audio quality in scenarios such as conferences, live streams, and recordings where unwanted background noise can degrade the listening experience.
+
+## Key Benefits
+
+- **Enhanced Audio Quality**: Eliminates unwanted noise, including background chatter, clicks, claps, barking, and other sudden audio disturbances, resulting in a more pleasant listening experience for your audience.
+
+- **Improved Clarity**: Ensures that the primary audio content remains prominent and intelligible by reducing distractions caused by ambient noise.
+
+- **Optimized Communication**: Facilitates seamless communication in conferences and live streams by minimizing disruptions caused by environmental factors, thereby enhancing the overall professionalism of the presentation.
+
+This is a guide to enabling and using the Noise Cancellation on 100ms.
+
+<StepsToc
+    parentId="noise-cancellation"
+    descriptions={[
+        "General information on 100ms Noise Cancellation.",
+        "Enabling the Noise Cancellation for your room(s).",
+        "Exploring platforms and methods of integration.",
+        "Activating and Managing Noise Cancellation."
+    ]}
+/>
+
+<StepsContainer id="noise-cancellation">
+
+### Getting Started
+100ms Noise Cancellation is powered by [krisp.ai](https://krisp.ai/),ensuring clear communication by filtering out background noise.
+
+<Note type='warning'>
+**IMPORTANT** <br/>
+This is an add-on paid feature, for details check [100ms pricing page](https://www.100ms.live/pricing/) 
+</Note>
+
+### Enabling the Noise Cancellation
+
+Noise Cancellation can be configured from the 100ms dashboard.
+
+#### Enabling the Noise Cancellation
+
+1. Navigate to a specific Template where you wish to enable the Noise Cancellation.
+2. Click on **‘Advanced Settings’** tab in the Template configuration.
+3. Enable ‘**Noise Cancellation**’.
+
+<video loop="true" autoplay="autoPlay" controls="false" id="vid" muted>
+    <source src="/docs/docs/v2/noise-cancellation/enable-nc.mp4" type="video/mp4" />
+</video>
+
+<Note>
+By default, Noise Cancellation is disbaled for all the peers in the session. To enable Noise Cancellation by default for all the peers check below section.
```

Comment:
This is incorrect actually.
2 updates here:
1. By default, noise cancellation is enabled for all peers and roles for all the new templates. This is something we had specifically come out with recently. You can verify it once.
2. Spelling of 'disbaled' to be updated to 'disabled'.